pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -139,6 +139,13 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
 	cephCluster.Status.Phase = condition
+	if status.Health.Status == "HEALTH_WARN" || status.Health.Status == "HEALTH_OK" {
```

Comment:
Will these values always be reported as 0 if it is HEALTH_ERR? There are some types of ceph errors that I wouldn't think should cause these not to be reported. For example, if there are only health errors for cephfs, the core cluster metrics wouldn't be affected. What if we update based on if `status.PgMap.TotalBytes != 0` instead of the health status directly?

Reply:
Why not check the content of `status.PgMap` instead?

Reply:
Done.